### PR TITLE
lottery result

### PR DESCRIPTION
**Pull request by WinPhyoEi**

### CODE REVIEW

**Comment by WinPhyoEi** [comment]
lotter result

File: package.json [hunk 1]
```diff
@@ -7,8 +7,9 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
     "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
+    "@testing-library/react": "^11.2.1",
     "@testing-library/user-event": "^12.1.10",
+    "bootstrap": "^4.5.3",
```
Comment (ayarhlaine) [inline package.json]: - remove bootstrap dependancies

File: src/components/Config/LotteryList.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        label :"ကျပ်သိန်း (၅၀၀၀) ဆုကြီး",
+        value :['45345','435345','3453453'],
+    },
+    {
+        label :"ကျပ်သိန်း (၅၀၀၀) ဆုကြီး",
+        value :['45345','435345','3453453']
+    }
+           
+];
```
Comment (ayarhlaine) [inline src/components/Config/LotteryList.jsx]: - rename file extension from `.jsx` to `.js`

File: src/components/LotteryList/LotteryList.css [hunk 3]
```diff
@@ -0,0 +1,9 @@
+p {
+    text-align: left;
+    font-size: 15px;
+    padding: 8px;
+}
+
+.LotteryList {
+    margin-left: 110px;
+}
```
Comment (ayarhlaine) [inline src/components/LotteryList/LotteryList.css]: - use Container from material UI

File: src/components/LotteryList/LotteryList.jsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import './LotteryList.css';
+import {lotteryList} from '../Config/LotteryList';
+const LotteryList = () => {
+    return (
+        <div className='LotteryList'> 
+          <h4> ထီဆုကြီးများ</h4>
+           {lotteryList.map( lottery => 
+              <p> {lottery}</p>
+           )}
```
Comment (ayarhlaine) [inline src/components/LotteryList/LotteryList.jsx]: - add key in lotteryList mapping
{lotteryList.map((lottery, key) => 
              <p key={key}> {lottery}</p>
 )}

File: src/components/LotteryResult/LotteryResult.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import './LotteryResult.css';
+const LotteryResult = () => {
+  console.log(lotteryResult)
+    return(
+      <div className="lottery-result">
+        <h4> ထွက်ရှိမည့် ထီဆုကြီးများ</h4>
+        <div className=" lottery-result-body">
+          {lotteryResult.map(( index,value ) =>         
+              <div key={value} className='lottery-result-button'  >{index.label}</div>
+          )}
```
Comment (ayarhlaine) [inline src/components/LotteryResult/LotteryResult.jsx]: - change code format
{lotteryResult.map(( lottery ,key ) =>         
              <div key={key} className='lottery-result-button'  >{lottery.label}</div>
 )}

File: src/components/LotteryResult/LotteryResult.jsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import {lotteryResult} from '../Config/LotteryList';
+import './LotteryResult.css';
+const LotteryResult = () => {
+  console.log(lotteryResult)
```
Comment (ayarhlaine) [inline src/components/LotteryResult/LotteryResult.jsx]: - remove console

File: package.json [hunk 7]
```diff
@@ -9,7 +9,6 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.2.1",
     "@testing-library/user-event": "^12.1.10",
-    "bootstrap": "^4.5.3",
```
Comment (ayarhlaine) [inline package.json]: Thanks you

File: src/components/LotteryResult/LotteryResult.jsx [hunk 8]
```diff
@@ -1,16 +1,28 @@
...
-          {lotteryResult.map(( index,value ) =>         
-              <div key={value} className='lottery-result-button'  >{index.label}</div>
-          )}
-        </div>
-      </div>  
+      <Container>
+        <h4> ထွက်ရှိမည့် ထီဆုကြီးများ</h4>   
+        <Grid container>          
+          {lotteryResult.map(( lottery,key ) =>           
+              <Grid item xs={12} sm={4} key={key} className='lottery-result-button'  >
```
Comment (ayarhlaine) [inline src/components/LotteryResult/LotteryResult.jsx:20]: Perfect

**Comment by ayarhlaine** [comment]
## UI Alignment Request
   Hello @WinPhyoEi ,
- Everything looks good, let me request one more thing on ui alignment, can you remove the margin from title, align?

Screen Shot 2020-11-20 at 10 40 29 PM

**Review by ayarhlaine — APPROVED** [review]
Great ! please merge this PR to main branch.

**Pull request merged**